### PR TITLE
Add `debounce` option to avoid flashing between channels

### PR DESCRIPTION
**Pull request by euaaaio**
Part of logux/logux#60

### CODE REVIEW

File: use-subscription/index.js [hunk 1]
```diff
@@ -48,14 +49,25 @@ function useSubscription (channels, opts = {}) {
...
+    if (debounce > 0) {
+      timeout = setTimeout(() => {
+        changeSubscribing(true)
+      }, debounce)
+    } else {
+      changeSubscribing(true)
+    }
+
     add(store, subscriptions).then(() => {
+      timeout && clearTimeout(timeout)
```
Comment (ai) [inline use-subscription/index.js:64]:
```suggestion
      if (timeout) clearTimeout(timeout)
```

File: use-subscription/index.js [hunk 2]
```diff
@@ -48,14 +49,25 @@ function useSubscription (channels, opts = {}) {
...
       if (!ignoreResponce) changeSubscribing(false)
     })
     return () => {
       ignoreResponce = true
       remove(store, subscriptions)
+      timeout && clearTimeout(timeout)
```
Comment (ai) [inline use-subscription/index.js:70]:
```suggestion
      if (timeout) clearTimeout(timeout)
```

Comment (ai) [reply use-subscription/index.js:70]: Should we move `clearTimeout` to some function which will also set `timeout = null`?

File: use-subscription/index.js [hunk 3]
```diff
@@ -48,14 +49,25 @@ function useSubscription (channels, opts = {}) {
     .join(' ')
 
   useEffect(() => {
+    let timeout
     let ignoreResponce = false
-    changeSubscribing(true)
+
+    if (debounce > 0) {
+      timeout = setTimeout(() => {
+        changeSubscribing(true)
```
Comment (ai) [inline use-subscription/index.js:57]: Will it set `isSubscribing=false` on initial loading?

File: use-subscription/index.js (hunk 2 above)
Comment (euaaaio) [reply use-subscription/index.js:70]: Oops 😬

File: use-subscription/index.js (hunk 3 above)
Comment (euaaaio) [reply use-subscription/index.js:57]: Yes: https://github.com/euaaaio/redux/blob/feat/use-subscription-debounce/use-subscription/index.test.ts#L286

**Pull request merged**